contracts/IVault.sol

```diff
@@ -5,9 +5,11 @@ import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import "@openzeppelin/contracts/interfaces/IERC4626.sol";
 
 interface IVault is IERC20Metadata, IERC4626 {
-    function getPerformance() external view returns (uint256);
+    
     function getMarket() external view returns (address);
     function getMarketAllowance() external view returns (uint256);
+    function getOwner() external view returns (address);
+    function getPerformance() external view returns (uint256);
```

Comment:
Actually, I thought we'd dropped `getPerformance()` given that anyone can calculate that?